http/parse.go

```diff
@@ -61,22 +61,28 @@ func parseRequest(r *http.Request, root *cmds.Command) (*cmds.Request, error) {
 	}
 
 	opts, stringArgs2 := parseOptions(r)
+	iopts := make(map[string]interface{}, len(opts))
 	optDefs, err := root.GetOptions(pth)
 	if err != nil {
 		return nil, err
 	}
 	for k, v := range opts {
+		iopts[k] = v
 		if optDef, ok := optDefs[k]; ok {
 			name := optDef.Names()[0]
 			if k != name {
-				opts[name] = v
-				delete(opts, k)
+				iopts[name] = v
+				delete(iopts, k)
```

Comment:
I'd clean this up a bit. Given this logic, there's no need to set the value in `iopts`, then delete it. We can just set it once.

Reply:
done